invenio_communities/assets/semantic-ui/js/invenio_communities/community/new.js

```diff
@@ -139,6 +140,7 @@ class CommunityCreateForm extends Component {
                   <TextField
                     fluid
                     fieldPath="metadata.title"
+                    onKeyDown={(e) => { e.key === "Enter" && e.preventDefault()}}
```

Comment:
`e.preventDefault()` will be called even if `e.key !== "Enter"`, is this intentional ?